registrations/models.py

```diff
@@ -0,0 +1,124 @@
+import uuid
+
+from django.contrib.postgres.fields import JSONField
+from django.contrib.auth.models import User
+from django.db import models
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from django.utils.encoding import python_2_unicode_compatible
+
+
+@python_2_unicode_compatible
+class Source(models.Model):
+    """ The source from which a registation originates.
+        The User foreignkey is used to identify the source based on the
+        user's api token.
+    """
+    AUTHORITY_CHOICES = (
+        ('patient', "Patient"),
+        ('advisor', "Trusted Advisor"),
+        ('hw_limited', "Health Worker Limited"),
+        ('hw_full', "Health Worker Full")
+    )
```

Comment:
I think these are fine. The authority is the same. There will be an additional Source record in the model, but that's not a source code change.